### PR TITLE
Add ST2110-30 receiver to the gstreamer and demo

### PR DESCRIPTION
**Pull request by joaofigueiredobisect**

### CODE REVIEW

File: cpp/libs/ossrf_gstreamer_api/lib/src/receiver/st2110_30_receiver_plugin.cpp [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    }
+
+    void stop() noexcept override
+    {
+        pipeline_.stop();
+        pipeline_ = {};
+    }
+};
+
+// TODO this function will need to receive an SDP and use the information in it to build the GST pipeline
```
Comment (massis08) [inline cpp/libs/ossrf_gstreamer_api/lib/src/receiver/st2110_30_receiver_plugin.cpp:101]: This function already receives information from the SDP, so this todo seams outdated

File: cpp/libs/ossrf_gstreamer_api/lib/src/receiver/st2110_30_receiver_plugin.cpp [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        BST_ENFORCE(gst_bin_add(GST_BIN(pipeline), source), "Failed adding udpsrc to the pipeline");
+
+        // Set udp source params
+        g_object_set(G_OBJECT(source), "address", s_.primary.source_ip_address.c_str(), NULL);
+        g_object_set(G_OBJECT(source), "auto-multicast", TRUE, NULL);
+        g_object_set(G_OBJECT(source), "port", s_.primary.source_port, NULL);
+        g_object_set(G_OBJECT(source), "multicast-iface", s_.primary.interface_name.c_str(), NULL);
+
+        // Create and set caps for udp source
+        GstCaps* caps = gst_caps_from_string("application/x-rtp, clock-rate=48000, channels=2");
```
Comment (massis08) [inline cpp/libs/ossrf_gstreamer_api/lib/src/receiver/st2110_30_receiver_plugin.cpp:59]: The `audio_info_t` has the information on the number of channels and clock rate to set on these caps, that is information parsed from the SDP

**Review by massis08 — APPROVED** [review]
LGTM

**Review by pedro-alves-ferreira — APPROVED** [review]
LGTM

**Pull request merged**